src/main/java/org/jvnet/hudson/update_center/Main.java

```diff
@@ -87,6 +93,15 @@
     @Option(name="-download",usage="Build mirrors.jenkins-ci.org layout")
     public File download = null;
 
+    @Option(name="-cache",usage="Cache target plugin")
```

Comment:
This documentation comment makes no sense to me.

This is a directory into which we download from the primary download location, i.e. the cache directory.

```suggestion
    @Option(name="-cache",usage="Cache directory used for caching plugin files")
```